packages/docusaurus/src/webpack/client.ts

```diff
@@ -56,16 +58,12 @@ export default function createClientConfig(
               ),
             );
 
-            stats.toJson('errors-only').errors.forEach((e) => {
```

Comment:
LogPlugin takes care of this, so it ends up printing errors twice if this is left in.